CONTRIBUTE.md

```diff
@@ -1,9 +1,7 @@
 Contribution
 ================
-Note, all contributions should be done on `dev` branch.
```

Comment:
This mention is important @Maks-s.

Reply:
It was important on the original Laverna project, but as @daed said on (Gitter)[https://gitter.im/encryptic-team/Lobby] : 
> violentmagician: daed for Encryptic i would suggest that we avoid a dev branch
> daed: I was thinking about the same thing. i guess it's not really necessary since it's mostly just me committing to it at this point in time. it would make things a bit simpler too. 

So far there's no dev branch so the instructions to use it are useless, but we can re-add them later if a dev branch is created